openwisp_monitoring/check/base/models.py

```diff
@@ -77,10 +75,9 @@ def perform_check(self, store=True):
 
 if app_settings.AUTO_PING:
```

Comment:
Since you are now connecting the post_save signal instead of using the 'receiver' decorator and you already check this condition in the _connect_signals method, I think it's safe to remove it now from here

Reply:
Fixed! :smile: